City-Bureau/city-scrapers-cle
# 🚫 Deprecate spider: Greater Cleveland Reentry Leadership Coalition

## What's this PR do?

Deletes our Greater Cleveland Reentry Leadership Coalition spider (aka. `cuya_reentry_leadership`).

## Why are we doing this?

The agency's website appears to have changed significantly and the spider is now broken. I can't find any public meetings or any kind of meeting page for this agency on [its website](https://hhs.cuyahogacounty.gov/divisions/detail/office-of-reentry).

## Steps to manually test

After installing the project using `pipenv`:

1. Activate the virtual environment:
```
pipenv shell
```
2. Run the spider:
```
scrapy crawl cuya_reentry_leadership -O test_output.csv
```
3. Monitor the stdout and ensure that the crawl proceeds without raising any errors. Pay attention to the final status report from scrapy.

4. Inspect `test_output.csv` to ensure the data looks valid. I suggest opening a few of the URLs under the source column of test_output.csv and comparing the data for the row with what you see on the page.

## Are there any smells or added technical debt to note?
